myapp/lib/tasks/maintenance.rake

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+desc 'start maintenance mode'
+task maintenance_go: :environment do
+  File.write('tmp/maintenance.txt', '')
+end
```

Comment:
[FYI]
It might be better to add 'logger' or 'puts'.

ex)
```ruby
desc 'start maintenance mode'
task maintenance_go: :environment do
  puts 'start maintenance mode !'

  File.write('tmp/maintenance.txt', '')

  puts 'maintenance mode completed !'
end
```